node/pkg/watchers/evm/blocks_by_timestamp.go

```diff
@@ -159,3 +159,9 @@ func (lhs Block) Cmp(rhs Block) int {
 
 	return 0
 }
+
+func (bts *BlocksByTimestamp) GetStats() (int, Block, Block) {
```

Comment:
is this still used anywhere? only found the definition here but no references

Reply:
Deleted it.